keps/sig-node/2413-seccomp-by-default/README.md

```diff
@@ -143,14 +143,33 @@ be extended to cover the new behavior if enabled.
 
 #### Alpha
 
-- [ ] Implement the new feature gate and kubelet configuration
-- [ ] Ensure proper tests are in place
-- [ ] Update documentation to make the feature visible
+- [x] Implement the new feature gate and kubelet configuration
+- [x] Ensure proper tests are in place
+- [x] Update documentation to make the feature visible
 
 #### Alpha to Beta Graduation
 
-- [ ] Enable the feature per default
-- [ ] No major bugs reported in the previous cycle
+- [x] Enable the feature per default
+- [x] Making the feature API aware
+- [x] No major bugs reported in the previous cycle
+
+##### API Awareness
+
+We add conversion methods to the API Server with the graduation of the feature
+to beta. The beta graduation does not mean that it's enabled by default, which
+got introduced in Kubernetes v1.24.
+
+The API Awareness allows us giving users direct feedback about the chosen seccomp
+profile as well as handling upgrade scenarios. If the feature is enabled, then
+created workloads not specifying any `securityContext.seccompProfile.type` will
+default to `RuntimeDefault`.
```

Comment:
The kubelet feature added in a previous release was opt-in. Will this be opt-in as well? If so, more than a feature flag is needed, since the opt-in goes beyond feature graduation. 

It also means this defaulting should only be done on pod creation, not in API defaulting for existing pods. This is because:
1. it depends on configuration, and conversion should be stateless
2. applying new defaults to existing fields in pods on API server upgrade triggers instant restarts of pods across nodes, which causes outages (https://github.com/kubernetes/kubernetes/issues/63814)